sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/api/Azure.Monitor.OpenTelemetry.Exporter.net6.0.cs

```diff
@@ -1,3 +1,15 @@
+namespace Azure.Monitor.OpenTelemetry.CustomEvents
+{
+    public sealed partial class CustomEventLogger : Azure.Monitor.OpenTelemetry.CustomEvents.ICustomEventLogger
+    {
+        public CustomEventLogger(Microsoft.Extensions.Logging.ILoggerFactory loggerFactory) { }
+        public void TrackEvent(string name, System.Collections.Generic.IReadOnlyList<System.Collections.Generic.KeyValuePair<string, object?>>? attributes = null) { }
+    }
+    public partial interface ICustomEventLogger
+    {
+        void TrackEvent(string name, System.Collections.Generic.IReadOnlyList<System.Collections.Generic.KeyValuePair<string, object?>>? attributes = null);
```

Comment:
Earlier API's did not support nullable, not sure it has changed. Please verify.

Reply:
We had defaults: https://learn.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.telemetryclient.trackevent?view=azure-dotnet#microsoft-applicationinsights-telemetryclient-trackevent(system-string-system-collections-generic-idictionary((system-string-system-string))-system-collections-generic-idictionary((system-string-system-double)))

Reply:
I meant the Azure SDK API's.

Reply:
I added a todo for following up on this.